src/js/initializers/initIpcListeners.ts

```diff
@@ -44,10 +44,6 @@ export default (store: Store) => {
     ipcRenderer.send(channel, getPersistable(store.getState()))
   })
 
-  ipcRenderer.on("getState", (event, channel) => {
-    ipcRenderer.send(channel, getPersistable(store.getState()))
-  })
-
```

Comment:
I am curious about the deletion here. It is used currently for gathering up all the window states when you click "Save Session for Testing Migrations".

Reply:
https://github.com/brimsec/brim/blob/master/src/js/electron/menu/appMenu.ts#L241

Reply:
Oh it's a duplicate, look at the lines right above it ;)

Reply:
Oh right 😅, thanks!